cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/ArmTemplateBuilder.java

```diff
@@ -77,6 +77,28 @@ public String build(String stackName, ArmCredentialView armCredentialView, ArmSt
             model.put("resourceGroupName", armUtils.getCustomResourceGroupName(network));
             model.put("existingVNETName", armUtils.getCustomNetworkId(network));
             model.put("existingSubnetName", armUtils.getCustomSubnetId(network));
+            model.put("userImageName", String.format("https://%s.blob.core.windows.net/%s/%s", rootDiskStorage, ArmStorage.IMAGES, imageName));
+            model.put("osDiskVhdName", String.format("https://%s.blob.core.windows.net/%s/%sosDisk",
+                    rootDiskStorage, armStorage.getDiskContainerName(cloudContext), stackName));
+            model.put("sshKeyPath", String.format("/home/%s/.ssh/authorized_keys", armCredentialView.getLoginUserName()));
+            model.put("ipConfigurationsAddress", String.format("'/frontendIPConfigurations/%sipcn", stackName));
+            model.put("ilbBackendAddress", String.format("/backendAddressPools/%sbapn", stackName));
+            model.put("loadBalancerAddress", String.format("Microsoft.Network/loadBalancers/%slb", stackName));
+            model.put("virtualNetworkAddress", String.format("Microsoft.Network/virtualNetworks/%s", stackName));
```

Comment:
The name prefix replaceable with stackname here: https://github.com/bioneyez/cloudbreak/blob/d3b5b53e9247325b655e2541f1591233987eccaf/cloud-arm/src/main/resources/templates/arm-v2.ftl#L109
And we don't need this any more: https://github.com/bioneyez/cloudbreak/blob/d3b5b53e9247325b655e2541f1591233987eccaf/cloud-arm/src/main/resources/templates/arm-v2.ftl#L39